validation/superchain-version_test.go

```diff
@@ -58,6 +58,7 @@ func TestContractVersions(t *testing.T) {
 		1750:      true, // mainnet/metal  L1CrossDomainMessengerProxy.version=1.4.1, https://github.com/ethereum-optimism/security-pod/issues/105
 		8453:      true, // mainnet/base
 		8866:      true, // mainnet/pontem L1CrossDomainMessengerProxy.version=1.4.1, https://github.com/ethereum-optimism/security-pod/issues/105
+		28882:     true, // sepolia/boba
```

Comment:
We're looking into the validation of the genesis test, but can you explain why you added an exception for this test?

Reply:
Since the fault-proof hasn't been deployed on Boba Sepolia L2 yet, we're unable to upgrade our OptimismPortal to v3.3.0, which this required by this test. However, we've set up the transactions for upgrading the other contracts through multisig.